modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -135,11 +135,12 @@ When the commands complete, you then have to:
 
 === php-libsodium
 
-NOTE: From PHP 7.2 onward, libsodium is part of PHP, so installing this extension is no longer necessary.
+NOTE: From PHP 7.2 onwards, libsodium is part of PHP, so installing this extension is no longer necessary.
```

Comment:
As the language of the docs is American English, onward and onwards are interchangeable, and have, effectively, the same meaning.

Reply:
we have used onwards on other places in this document too, therefore keeping it inline